README.md

```diff
@@ -34,7 +34,7 @@ Follow the below instructions to deploy your app using the Heroku button and Nod
 ### Add the starter app to LIFF
 
 1. Follow the instructions on the page [Adding a LIFF app](https://developers.line.biz/en/docs/liff/registering-liff-apps/). 
-2. Take a note of your LIFF ID, because you'll need it for the next part. The LIFF ID is the final part of the LIFF URL shown in the console: `line://app/{liffId}`.
+2. Take a note of your LIFF ID, because you'll need it for the next part. The LIFF ID is the final part of the **LIFF URL** shown in the console: `https://app/{liffId}`.
```

Comment:
As is: `https://app/{liffId}`
To be: `https://liff.line.me/{liffId}`

Reply:
Hi @smwilk, I wasn't sure it was always the case, I was assuming that you had the same documentation for external/in-house development with different consoles/APIs. Thanks a lot!

Reply:
You are absolutely right, this is a change that was made recently. Please see [this news article](https://developers.line.biz/en/news/2020/03/25/line-url-scheme-deprecation/). Could you please update your code as specified [here](https://github.com/line/line-liff-v2-starter/pull/12#discussion_r411954646), and I can merge this PR.